googletest/include/gtest/gtest-printers.h

```diff
@@ -983,7 +983,7 @@ struct TuplePolicy {
 
   template <size_t I>
   static typename AddReference<const typename ::std::tr1::tuple_element<
-      static_cast<int>(I), Tuple>::type>::type
+      I, Tuple>::type>::type
```

Comment:
As I explained in https://github.com/google/googletest/pull/1815 - that change was made for a reason. We can not just roll it back. You may want to consider using macros to only do this when this version of MSVC is used .